.development/googlesheets_dataclean.R

```diff
@@ -4,36 +4,54 @@ library(janitor)
 library(tidyr)
 library(googlesheets4)
 
-sheet_url <- "https://docs.google.com/spreadsheets/d/1usk9Q-5lA4bL_z6KXpUohc_2x_KhDgLxtm-YEtim_yk/"
+sheet_url <- "https://docs.google.com/spreadsheets/d/1007RLMHSukSJ5OfCcDJdnJW5QMZyS2P-81fe7utCZwk/"
```

Comment:
There are several times we are explicitly calling the name of the GSheet. A better approach is to specify this in an `.Renviron` file and call it from there. This way, if the GSheet link changes, we won't need to change it in each of the scripts that call it, we just need to change it in the `.Renvirn` file. 

Nothing to worry about now. Probably something to pick up in a different PR.